packages/vitest/src/types/config.ts

```diff
@@ -112,6 +113,8 @@ interface DepsOptions {
    * And does not support HMR on reload.
    *
    * Typically, packages under `node_modules` are externalized.
+   *
+   * @deprecated Use `server.deps.external` instead
```

Comment:
`server.` is only if users rely on vite-node directly. We need to add a note to use `optimizer` instead.